smart-contracts/advanced/aggregated-deal-making.md

```diff
@@ -55,12 +55,12 @@ To request for aggregation and PoDSI off-chain, developers interact with an aggr
 
 On-chain aggregation and PoDSI requests go through aggregator oracle smart contracts:
 
-1. The client [prepares the data](../../storage-providers/filecoin-deals/storage-deals/#data-preparation) and generates the sub-piece CID, known as pCID (CommPc). Here is an easy [data preparation tool](https://data.lighthouse.storage/) by [lighthouse.storage](https://lighthouse.storage).
+1. The client [prepares the data](https://github.com/filecoin-project/filecoin-docs/blob/main/storage-providers/filecoin-deals/storage-deals.md#data-preparation) and generates the sub-piece CID, known as pCID (CommPc). Here is an easy [data preparation tool](https://data.lighthouse.storage/) by [lighthouse.storage](https://lighthouse.storage).
```

Comment:
Is there a reason these need to be absolute now?

Reply:
The links are currently not working on [filecoin docs](https://docs.filecoin.io/smart-contracts/advanced/aggregated-deal-making), so I changed them to absolute. I haven't investigated further, but I can dig in?

Reply:
Hmm .. I not sure why not .. the path seems correct. 😕

Reply:
Thank you, fixed!